src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/SetAzureRmAnalysisServicesServer.cs

```diff
@@ -71,6 +72,27 @@ public class SetAzureAnalysisServicesServer : AnalysisServicesCmdletBase
             HelpMessage = "The switch to turn off backup of the server.")]
         public SwitchParameter DisableBackup { get; set; }
 
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 6, Mandatory = false,
+            HelpMessage = "The replica count of readonly pool")]
+        [ValidateRange(0, 7)]
+        public int ReadonlyReplicaCount
+        {
+            get { return _readOnlyReplicaCount; }
+            set { _readOnlyReplicaCount = value; }
+        }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 7, Mandatory = false,
+            HelpMessage = "The default connection mode to query server")]
+        [ValidateSet("All", "Readonly", IgnoreCase = true)]
+        public string DefaultConnectionMode { get; set; }
+
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 8, Mandatory = false,
+            HelpMessage = "A json string to indicate the firewall setting")]
```

Comment:
It's a json string? Should be the same message as in New

Reply:
done

Reply:
done